ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -2330,16 +2330,16 @@ periodics:
               cluster_profile: openstack-vexxhost
               workflow: openshift-e2e-openstack-ipi
               test:
-              - as: e2e-test
-                commands: sleep $(CLUSTER_DURATION) & wait
-                from_image:
-                  namespace: origin
-                  name: centos
-                  tag: '8'
-                resources:
-                  requests:
-                    cpu: 100m
-                    memory: 200Mi
+              - ref: clusterbot-wait
+          - as: launch-kuryr
+            steps:
+              cluster_profile: openstack-kuryr
+              env:
```

Comment:
Please add the `CLUSTER_DURATION` env var to this test as well as the other launch job as seen here: https://github.com/openshift/release/blob/1b9fb8e44724b757dd26c75320967ed2b1d871da/ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml#L262:L263. This allows the ci-chat-bot to manage the `CLUSTER_DURATION` if necessary.

Reply:
done, thanks